apps/site/lib/site_web/templates/event/_expandable_month.html.eex

```diff
@@ -0,0 +1,28 @@
+<section id="<%= @month_number %>-<%= @year %>" class="m-event-list__month <%= if @month_number == @month, do: 'm-event-list__month--active' %>">
+  <% header_id = "header-#{@month_number}" %>
+  <% panel_id = "panel-#{@month_number}" %>
+
+  <button class="c-expandable-block__link sticky-top sticky-month"
```

Comment:
I think the `sticky-top sticky-month` classes need to be with the `h2`/`.m-event-list__month-header` on line 12 below. Hopefully that will fix the two regressions I noticed:

* The h2 not getting the `stuck` attribute when the heading sticks to the top of the page, thereby no drop shadow below the header
* The sticky header not hiding/revealing when scrolling up/down (this is on mobile only).

Reply:
I think the shadow should be back with the latest amendment :)

Reply:
@alicia-mbta I don't see any update?

Reply:
@thecristen Moving the sticky classes to the `h2`'s didn't work (the stickiness was being ignored), so I modified the selector instead:
https://github.com/mbta/dotcom/blob/277576ab5d5119dfa73057afa4cb6f0771c8c135/apps/site/assets/js/event-page-setup.js#L130-L132
and the nesting in the CSS property:
https://github.com/mbta/dotcom/blob/277576ab5d5119dfa73057afa4cb6f0771c8c135/apps/site/assets/css/_events.scss#L55-L60